plugin/yt_dlp_plugins/extractor/getpot_bgutil_http.py

```diff
@@ -36,18 +36,27 @@ def _real_validate_get_pot(
                     f'{base_url}/ping', extensions={'timeout': self._GET_VSN_TIMEOUT}, proxies={'all': None})))
             except TransportError as e:
                 # the server may be down
-                self._warn_and_raise(
-                    f'Error reaching GET /ping (caused by {e.__class__.__name__})')
+                script_path_provided = self._get_config_setting(
+                    'getpot_bgutil_script', default=None) is not None
+                warning_base = f'Error reaching GET {base_url}/ping (caused by {e.__class__.__name__}). '
+                if script_path_provided:  # server down is expected, log info
+                    self._info_and_raise(warning_base + 'This is expected if you are using the script method.')
+                else:
+                    self._warn_and_raise(warning_base + f'Otherwise, make sure that the server is reachable at {base_url}.')
```

Comment:
```suggestion
                    self._warn_and_raise(warning_base + f'Please make sure that the server is reachable at {base_url}.')
```
forgot to remove the `otherwise`